esp/esp/web/util/latex.py

```diff
@@ -140,59 +140,40 @@ def gen_latex(texcode, type='pdf', landscape=False, remove_files=False, stdout=P
     
     #   Set latex options
     latex_options = ['-interaction', 'nonstopmode', '-halt-on-error']
-    #   Set dvips options
-    dvips_options = ['-t', 'letter']
-    if landscape:
-        dvips_options = ['-t', 'letter,landscape']
```

Comment:
Does pdflatex take care of the orientation for us? If so, can you remove the landscape argument to this function?